docs/sources/tempo/release-notes/v2-3.md

```diff
@@ -0,0 +1,201 @@
+---
+title: Version 2.3 release notes
+menuTitle: V2.3
+description: Release notes for Grafana Tempo 2.3
+weight: 45
+---
+
+# Version 2.3 release notes
+
+The Tempo team is pleased to announce the release of Tempo 2.3.
+
+This release gives you:
+
+* New [structural operators]({{< relref "../traceql#experimental-structural" >}}) in TraceQL:
+    * Ancestor (`<<`)
+    * Parent (`<`)
+    * Not-child (`!>`) (experimental)
+    * not parent (`!<`) (experimental)
+    * not siblings (`!~`) (experimental)
+
+* TraceQL support for [searching quoted attributes]({{< relref "../traceql#experimental-structural" >}}) (for example, `{ span."name with spaces" = "foo" }`).
+* [Dedicated attribute columns]({{< relref "../operations/dedicated_columns" >}}) for improving query performance on your most frequently queried attributes
+
+Tempo 2.3 makes [vParquet3]({{< relref "../configuration/parquet" >}}), a Parquet version designed to be more compatible with other Parquet implementations, available as a production-ready option.
+This block format is required for using structural operators and improves query performance relative to previous formats.
```

Comment:
2.2 does structural operators but 2.3 is faster

Reply:
```suggestion
This block format improves query performance relative to previous formats. Structural operators require vParquet3.
```

Reply:
all structural operators (including the new ones) work on both vp2 and vp3. there are, however, optimizations in 2.3 which make them a bit faster (with more to come).